packages/blockchain-wallet-v4-frontend/src/data/goals/sagas.js

```diff
@@ -86,6 +87,17 @@ export default ({ api }) => {
     yield put(actions.router.push('/login'))
   }
 
+  const defineSendXlmGoal = function * (pathname, search) {
+    // /#/open/xlm?address={address}&amount={amount}
```

Comment:
No amount yet

Reply:
I see memo in the code so should the format be: `/#/open/xlm?address={address}&amount={amount}&memo={memo}`?

Reply:
nvm, i see it in the code right below this comment.

Reply:
You're right, dont think I included memo

Reply:
That format should work and I can make the appropriate change